src/main/java/de/tarent/maven/plugins/pkg/TargetConfiguration.java

```diff
@@ -285,6 +285,13 @@ public TargetConfiguration(String target) {
 	private String defaultDistro;
 
 	/**
+	 * Make the defaultDependencyLine configurable so we can override
+	 * it, for wheezy when we want be able to use openjdk-7-jre-headless
```

Comment:
typos:
- remove the comma
- remove "be able" (or reword the sentence)
- append a period (".")